fitbenchmarking/fitbenchmarking
# Optional plotting

#### Description of Work

Fixes #378 

Adds option to make plots

#### Testing Instructions

1. Check option is correctly set and executed

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
